examples/PublishHelloWorldCAN/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 2.8.12)
+if (${CMAKE_VERSION} VERSION_GREATER 3.0)
+    cmake_policy(SET CMP0048 NEW)
+endif()
+
+project(PublishHelloWorldClientCAN C)
+
+if(NOT UCLIENT_BUILD_EXAMPLES)
+    find_package(microxrcedds_client REQUIRED)
+endif()
+
+if(NOT UCLIENT_PROFILE_UDP)
```

Comment:
This label should be UCLIENT_PROFILE_CAN

Reply:
Fixed 85039e28d79cfd30d4ad32e89451a0c584f145b6